index.html

```diff
@@ -4255,6 +4381,28 @@ <h2>
             </ol>
           </li>
           <li>If the <a data-lt=
+          "PaymentOptions.requestPayerAddress">requestPayerAddress</a> value of
+          <var>request</var>.<a>[[\options]]</a> is false, then set the
+          <a data-lt="PaymentResponse.payerAddress">payerAddress</a> attribute
+          value of <var>response</var> to null. Otherwise:
+            <ol>
+              <li>Let <var>redactList</var> be the empty <a>list</a>.
+              </li>
+              <li>Let <var>payerAddress</var> be the result of <a>create a
+              <code>PaymentAddress</code> from user-provided input</a> with
+              <var>redactList</var>.
+              </li>
+              <li>Set the <a data-lt=
+              "PaymentResponse.payerAddress">payerAddress</a> attribute value
+              of <var>response</var> to <var>payerAddress</var>.
+              </li>
+              <li>Set the <a data-lt=
+              "PaymentRequest.payerAddress">payerAddress</a> attribute value of
+              <var>request</var> to <var>payerAddress</var>.
```

Comment:
It'll be worth testing that UAs create distinct objects here and change payerAddress to the new PaymentAddress object. That is, they can't mutate the existing object; per this spec they have to create a new one.

Reply:
Will make sure to test that.